crates/deskulpt-core/Cargo.toml

```
@@ -28,7 +28,11 @@ serde_with                   = { workspace = true }
 serialize-to-javascript      = { workspace = true }
 specta                       = { workspace = true, features = ["derive", "function", "serde_json"] }
 tauri-plugin-global-shortcut = { workspace = true }
+time                         = { workspace = true }
 tokio                        = { workspace = true }
+tracing                      = { workspace = true }
+tracing-appender             = { workspace = true }
+tracing-subscriber           = { workspace = true }
```

```suggestion
tracing-subscriber           = { workspace = true, features = ["env-filter", "time"] }
```

This way we won't need to depend on `time`.